R/RMeCabDF.R

```diff
@@ -33,13 +33,13 @@ RMeCabDF <-
 ##       stop("the second argument must be integer")
 ##     }
 ## #    gc()
-    kekka <- list(length(dataf[,coln]))
-    for(i in 1:length(dataf[,coln])){
-      if( is.na(dataf[i,coln])  ||  dataf[i, coln] == ""){## if( !is.factor(dataf[i,coln]) || is.na(dataf[i,coln])  ||  dataf[i, coln] == ""){
+    kekka <- list(length(dataf[,coln,drop=T]))
```

Comment:
ご提案ありがとうございます。
```suggestion
    kekka <- list(length(dataf[,coln,drop=TRUE]))
```
TRUEとフル表記にしたいと思います。